components/link-social/css/index.scss

```diff
@@ -4,17 +4,20 @@
  */
 
 .utrecht-link-social {
-  --utrecht-icon-color: var(--utrecht-color-white);
+  --utrecht-icon-color: var(--utrecht-link-social-icon-color);
+  --utrecht-icon-size: var(--utrecht-link-social-icon-size);
```

Comment:
Maybe we can simplify this to: `--utrecht-icon-size: var(--utrecht-link-social-size)`